assignment/api/src/java/org/sakaiproject/assignment/api/AssignmentService.java

```diff
@@ -817,6 +818,16 @@ public String getDeepLinkWithPermissions(String context, String assignmentId, bo
      */
     public List<MultiGroupRecord> checkSubmissionForUsersInMultipleGroups(String siteId, Group submissionGroup, Collection<Group> asnGroups);
 
+    public boolean correctTime(String timeSheet);
+
+    public AssignmentSubmissionSubmitter getSubmissionSubmitter(String submitterId) throws PermissionException;
+
+    public AssignmentTimeSheet getTimeSheet(String timeSheetId) throws PermissionException;
+	
+    public void addAssignmentTimeSheet(AssignmentTimeSheet timeSheet, String context) throws PermissionException;
```

Comment:
update method signature addTimeSheet(AssignmentTimeSheet timeSheet, String siteId)

Reply:
context renamed to siteId

Since there's already an addTimeSheet method, this method has been renamed to 'setTimeSheet'